templates/master.yaml.tmpl

```diff
@@ -1695,6 +1725,24 @@ storage:
       mode: 0644
       contents:
         inline: |
+          ## giantswarm-clusterapi cluster role for readonly API access
+          apiVersion: rbac.authorization.k8s.io/v1
+          kind: ClusterRole
+          metadata:
+            name: giantswarm-clusterapi-readonly
+            namespace: giantswarm-clusterapi
+            labels:
+              scope: control-plane
+              type: external
+              level: readonly
+          rules:
+          - apiGroups:
+            - ""
+            resources:
+            - nodes
```

Comment:
I guess this is start and will be adjusted later

Reply:
Yep, this is just a base for tests